R/summary_functions.R

```diff
@@ -4,8 +4,9 @@
 #' @export
 percent_coverage <- function(coverage_result){
   coverage_data <- as.data.frame(coverage_result)
-  num_lines <- nrow(coverage_data)
-  num_not_zero <- sum(coverage_data$value != 0) # how many lines have a value that is not zero?
+  num_lines <- length(unique(coverage_data$first_line))
```

Comment:
No thinking about it more lets add a parameter as you suggested and have
percent coverage by line be the default.

On Tue, Jan 13, 2015 at 11:30 AM, Karl Forner <notifications@github.com>
wrote:

> In R/summary_functions.R
> <https://github.com/jimhester/covr/pull/30#discussion-diff-22873736>:
>
> > @@ -4,8 +4,9 @@
> >  #' @export
> >  percent_coverage <- function(coverage_result){
> >    coverage_data <- as.data.frame(coverage_result)
> > -  num_lines <- nrow(coverage_data)
> > -  num_not_zero <- sum(coverage_data$value != 0) # how many lines have a value that is not zero?
> > +  num_lines <- length(unique(coverage_data$first_line))
>
> so what do you want me to do ? restore your original functions and add new
> ones ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jimhester/covr/pull/30/files#r22873736>.
>